src/components/OrderDetails.js

```diff
@@ -0,0 +1,199 @@
+            button: "btn btn-primary mt-1 "
+        }
+
+        return (
+            <div className={bootstrapCss} >
+                <div className={"w-25 m-1 container "}>
+                    <div className={customCSS.wrapper}>
+                        <h1>Order details</h1>
+
+                        <form onSubmit={this.handleSubmit} >
+                            <label>First name:
+                                <input
+                                    type="text"
+                                    name="firstName"
+                                    onChange={this.handleChange}
+                                    className={customCSS.inputs}
+                                />
+                            </label>
+                            <label>Last name:
+                                <input
+                                    type="text"
+                                    name="lastName"
+                                    onChange={this.handleChange}
+                                    className={customCSS.inputs}
+                                />
+                            </label>
+                            <label >Your address:
+                                <input
+                                    type="text"
+                                    name="address"
+                                    onChange={this.handleChange}
+                                    className={customCSS.inputs}
+                                />
+                            </label>
+                            <label>Your email:
+                                <input
+                                    type="email"
+                                    name="email"
+                                    onChange={this.handleChange}
+                                    className={customCSS.inputs}
+                                />
+                            </label>
+                            <label>Your phone number:
+                                <input
+                                    type="tel"
+                                    name="telephone"
+                                    pattern="^[+]?[(]{0,1}[0-9]{1,4}[)]{0,1}[-\s\./0-9]*$"
+                                    onChange={this.handleChange}
+                                    className={customCSS.inputs}
+                                />
+                            </label>
+                            <label>Your message to the pizzaboy:
+                                <input
+                                    type="text"
+                                    name="notes"
+                                    placeholder="The doorbell doesn't work, call me instead"
+                                    onChange={this.handleChange}
+                                    className={customCSS.inputs}
+                                />
+                            </label>
+                            <br />
```

Comment:
Replace input field with a map solution. Create an array which contains object with following parameters: labelName, inputType, inputName. Map them to Memo Functional component, which contains a label and input.